WireAPI/Sources/WireAPI/APIs/UpdateEventsAPI/Event decoding/FeatureConfig/ConferenceCallingFeatureConfigDecoder.swift

```diff
@@ -28,7 +28,10 @@ struct ConferenceCallingFeatureConfigDecoder {
             forKey: .payload
         )
 
-        return ConferenceCallingFeatureConfig(status: payload.status)
+        return ConferenceCallingFeatureConfig(
+            status: payload.status,
+            useSFTForOneToOneCalls: nil
```

Comment:
I suppose that we should also parse the `useSFTForOneToOneCalls` for the update event too.

Reply:
We could introduce a dedicated `Payload` with a config to properly decode the `useSFTForOneToOneCalls` property however since these are not versioned I assume we always receive a value for it ?

Reply:
`useSFTForOneToOneCalls` was only introduce in v6, btw

Reply:
as per backend team, this property is not part of the decodable model, so we're just assigning a `false` default value. (optional has been removed)